prime-angular-frontend/src/app/modules/enrolment/pages/obo-sites-page/obo-sites-page.component.ts

```diff
@@ -211,13 +217,38 @@ export class OboSitesPageComponent extends BaseEnrolmentProfilePage implements O
         this.setOboSites();
       });
     } else {
-      //for renewal, pull the enrolment from enrolment service instead
+
       this.enrolment = this.enrolmentService.enrolment;
-      this.enrolmentFormStateService.patchOboSitesForm(this.enrolment.oboSites);
+      if (this.lastUpdatedDatetime.value == null) {
+        //for renewal, pull the enrolment from enrolment service instead
```

Comment:
Is it more accurate to leave out this comment and for the `else` clause, say "If the enrollee updated the OBO Sites form, use information from this form (the latest information) instead"?

Reply:
Agree, and it is updated.